CHANGELOG.md

```diff
@@ -77,6 +78,12 @@
 
 [issue-2567]: https://github.com/googleapis/google-cloud-cpp/issues/2567
 
+### IAM:
+
+**BREAKING CHANGES**:
+* The `experimental-iam` target has been replaced with the target `iam`. Current
```

Comment:
I believe this only applies to bazel builds, since the `experimental-iam` target still exists in cmake. You could clarify that here. Or, you could add the `experimental-{iam,bigquery}` targets back to //BUILD so there is no breaking change this release, and the bazel and cmake targets would remain in sync. Then after next weeks release you could then remove the `experimental-{iam,bigquery}` targets from BOTH bazel and cmake, and add the **BREAKING CHANGES** doc for the following month's release. I think that latter option is what I'd do, but I think I've caused you enough headache, so I'll approve and let you decide how you want to proceed :-) Thanks.